src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arraySorceString = sourceString
+    .split(';')
+    .map((string) => string.trim().split(':'));
+
+  const filteredSorceString = arraySorceString
+    .filter((innerArr) => innerArr.length === 2)
+    .map((element) => element.map((item) => item.trim()));
```

Comment:
Consider renaming the variable 'arraySorceString' to a more descriptive name that reflects its purpose, such as 'styleDeclarations'. Using names like 'array' can decrease code readability.